packages/brand-components/src/Avatar/__tests__/avatar-test.tsx

```diff
@@ -0,0 +1,38 @@
+import { mount } from 'enzyme';
+import React from 'react';
+
+import { Avatar } from '..';
+
+describe('Avatar', () => {
+  it('when an "alt" prop is passed, an "alt" attribute is added to the <img/>', () => {
+    const wrapper = mount(<Avatar src="" alt="alt" />);
+
+    expect(wrapper.find('img[alt="alt"]').length).toEqual(1);
```

Comment:
Have you considered this?

```suggestion
    expect(wrapper.find('img')).toHaveProp('alt', alt);
```

A little easier to read and a bit better error message on failure.

Reply:
Causes type error:
```
Property 'toHaveProp' does not exist on type 'JestMatchersShape<Matchers<void, ReactWrapper<HTMLAttributes, any, Component<{}, {}, any>>>, Matchers<Promise<void>, ReactWrapper<HTMLAttributes, any, Component<{}, {}, any>>>>'.ts(2339)
```
Looking into it.

Reply:
Will use https://github.com/Codecademy/client-modules/pull/631/commits/7947046e29034cc8109a6d0b5f8efecf02e62e2a